_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -256,6 +269,12 @@ private function getContentComposite($content) {
     return $item;
   }
 
+  /**
+   * Please update this function description..
+   *
+   * @return array
+   *   Please update this return description.
+   */
   private function getContentCompositeMultiple($contents) {
```

Comment:
This is just like getContentComposite() but pass multiple posts and formats and returns those multiple posts